### DISCUSSION TITLE
GitHub Education application

### DISCUSSION THREAD

**Discussion by neridisoq** [community/community#60645] [GitHub Education]
### Hi everyone,

I tried to apply many times, and I finally succeeded (about 8th tries?). I saw I need to wait 6days to check my applications but I got 7 fail mails not “request success” mail. Is it common? Or do I need to remake the application?

**Comment by OshekharO** [comment]
You can contact the GitHub Education support team by visiting the GitHub Education website and navigating to their support or contact page. They will be able to assist you further with any questions or concerns you have regarding your application.

PS: You need to fill again.

**Comment by samus-aran** [comment]
Hi @helgisnw 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
